### PR TITLE
Feauture/expanding buttons

### PR DESCRIPTION
**Pull request by MsweliP**
UI chnages

### CODE REVIEW

File: waterlog/src/Components/BtnResolve.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchSegmentsLeaksResolve } from '../actions/SegmentResolveActions';
+import { Globals } from './../Globals';
+
+class BtnResolve extends Component {
```
Comment (kimibeyl) [inline waterlog/src/Components/BtnResolve.js]: Just fix your indentation

File: waterlog/src/Components/BtnResolve.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+				RESOLVE
+			</button>
+		</div>
+    );
+  }
+
+  resolve(leaksObject) {
+		leaksObject = this.props.leaksResolves;
+		leaksObject.resolvedStatus = "resolved";
+		return fetch(`${Globals.API_URL}/api/segmentleaks/${leaksObject.id}`, {
```
Comment (kimibeyl) [inline waterlog/src/Components/BtnResolve.js]: It is better to have all your api calls externally in seperate file

File: waterlog/src/Components/BtnResolve.js (hunk 1 above)
Comment (MsweliP) [reply waterlog/src/Components/BtnResolve.js]: Done

Comment (kimibeyl) [reply waterlog/src/Components/BtnResolve.js]: Sorry I meant less indentation all the functions needs to look the same in terms of tabbing look at the Map.js component

Comment (MsweliP) [reply waterlog/src/Components/BtnResolve.js]: Done please review again

Comment (kimibeyl) [reply waterlog/src/Components/BtnResolve.js]: Are you sure?

**Review by kimibeyl — DISMISSED** [review]

File: waterlog/src/Components/BtnResolve.js (hunk 1 above)
Comment (MsweliP) [reply waterlog/src/Components/BtnResolve.js]: Yes I'm sure

File: waterlog/src/Components/BtnResolve.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+			cache: "no-cache",
+			credentials: "same-origin",
+			headers: {
+				"Content-Type": "application/json",
+			},
+			redirect: "follow",
+			referrer: "no-referrer",
+			body: JSON.stringify(leaksObject),
+		})
+			.then(response => response.json());
```
Comment (FredJEhlers) [inline waterlog/src/Components/BtnResolve.js]: `.then(response => response.json());` needs to be on line 54 right after the `})`

Comment (MsweliP) [reply waterlog/src/Components/BtnResolve.js]: Done please re-check again

File: waterlog/src/Components/BtnResolve.js (hunk 1 above)
Comment (FredJEhlers) [reply waterlog/src/Components/BtnResolve.js]: @MsweliP What Kimi is pointing out. Your project has used tab 2 in other files [eg:](https://github.com/Hunter27/WaterLog-Front-end/blob/master/waterlog/src/Components/Tank.js) and now here you are using tab 4

**Review by FredJEhlers — APPROVED** [review]

File: waterlog/src/Components/BtnResolve.js (hunk 2 above)
Comment (GeromeSchutte) [reply waterlog/src/Components/BtnResolve.js]: from a consistency point of view, you have some actions in `segmentResolveActions` and some in this file. Please move all your API calls to an action file.

**Review by GeromeSchutte — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/SegmentLeak.js [hunk 4]
```diff
@@ -72,13 +76,13 @@ class SegmentLeak extends Component {
...
 			<div>
-				<div className={`leakInfo ${parseInt(alert.status) == 2 ? alert.severity : 'leak-resolved'}`}>
-					<h2>{`${alert.entityName} ${alert.entityId} ${alert.entityType}`}</h2>
-					<p>({alert.severity})</p>
-					<h1>R {alert.cost.toFixed(2)}</h1> 
-					<p>is being lost per hour!</p>
-					<p>Loosing {alert.litresPerHour.toFixed(0)}&#x2113; per hour</p>
-					<p>no leak would be 0&#x2113; per hour</p>
+				<div>
+					<h2 className={this.getSeverityClass(alert.severity)}>{`${alert.entityName} ${alert.entityId} ${alert.entityType}`}</h2>
```
Comment (GeromeSchutte) [inline waterlog/src/Components/SegmentLeak.js]: call the function once, assign its value to a variable and assign that variable to the class

File: waterlog/src/Stylesheets/_progressBar.scss [hunk 5]
```diff
@@ -5,15 +5,44 @@
   margin: 0 10px;
 }
 
-.filler {
-  background-color: red;
+.filler-low {
+  background-color: #FFEA00 !important;
```
Comment (GeromeSchutte) [inline waterlog/src/Stylesheets/_progressBar.scss]: don't use important unless you are overriding inline styles used by component libraries.

**Review by GeromeSchutte — CHANGES_REQUESTED** [review]

File: waterlog/src/Stylesheets/_progressBar.scss (hunk 5 above)
Comment (MsweliP) [reply waterlog/src/Stylesheets/_progressBar.scss]: Done please re-check

File: waterlog/src/Components/SegmentLeak.js (hunk 4 above)
Comment (MsweliP) [reply waterlog/src/Components/SegmentLeak.js]: Done please re-check

**Review by GeromeSchutte — APPROVED** [review]

**Pull request merged**